README.md

```diff
@@ -42,23 +42,32 @@ To install Jenkins for the DCOS, perform the following steps.
 
 Jenkins should now be available at <http://dcos-master/service/jenkins>. See [Getting Started][getting-started] for more in-depth instructions and configuration options.
 
+## Releasing
+To release a new version of this package:
+
+  1. Update [the Jenkins conf](jenkins-conf) to reference the next release of the [jenkins-dind](jenkins-dind) Docker image.
+  2. Tag the commit on master that you want to be released.
+  3. Once [the build](teamcity-build) has successfully completed, submit a new pull request against [the Universe](universe) referencing the new tag.
```

Comment:
These named links should use square brackets instead of parens. Otherwise, looks good!